entries/u/udemy.com.json

```
@@ -5,7 +5,6 @@
       "email"
     ],
     "documentation": "https://support.udemy.com/hc/en-us/articles/1500009156921",
```

This page is probably more applicable then.
```suggestion
    "documentation": "https://support.udemy.com/hc/en-us/articles/4420349685015",
```

---

Good call, I didn't check it!